robottelo/config/__init__.py

```diff
@@ -49,8 +48,6 @@
 settings = SettingsNodeWrapper(settings_proxy)
 settings.configure_nailgun()
 settings.configure_airgun()
-settings.configure_logging()
```

Comment:
@JacobCallahan This makes your dynaconf end game PR just a little bit smaller :)